contentService/src/main/resources/application.properties

```diff
@@ -1,10 +1,12 @@
 server.port = 8083
 spring.application.name = contentService
 
-spring.datasource.url=jdbc:mysql://localhost:3306/testdb
-spring.datasource.username=root
-spring.datasource.password=12345
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
-spring.jpa.hibernate.ddl-auto=update
-spring.jpa.properties.hibernate.show_sql=true
-spring.jpa.properties.hibernate.format_sql=true
\ No newline at end of file
+spring.config.import=aws-parameterstore:/config/contentService/
+
+#spring.datasource.url=${spring.datasource.url}
```

Comment:
If you don't want, please remove this don't use comment here.

Reply:
Removed